CHANGELOG.md

```diff
@@ -1,5 +1,208 @@
 # Changelog
 
+## 5.0.0.rc1
+
+This is a major Teleport release with a focus on new features, functionality, and
+bug fixes. It’s a substantial release and users can review [5.0 closed issues](https://github.com/gravitational/teleport/milestone/39?closed=1)
+on Github for details of all items.
+
+#### New Features
+Teleport 5.0 is a major version and an introduction to Teleports Unified Access
+Plane. Going beyond just SSH access, with first class Kubernetes support
+and the new addition of Application Support.
+
+##### Teleport Application Access
+Teleport can be used to provide secure access to Applications. We've
```

Comment:
web apps. This is not a marketing release, but a note for engineers, can we keep it close to devs language?

Reply:
Fixed